blockstack/blockstack-app-generator
# Vue.JS generator

Closes #12 

I was able to create a hello blockstack generator using Vue.js.

Steps to test:
make sure you have generator-blockstack installed as well as yoeman,
pull down these changes to your local machine,
create a new test folder anywhere and cd into it,
run npm link,
they run yo blockstack:vue

This should locally link the generator to the yeoman command. Once this is merged to master, please make sure it and the react generator are added to the yeoman commands globally since the community has pointed out that the commands are not quite working as shown in the readme.
